bin/goto

```diff
@@ -10,5 +10,15 @@ if [ "$1" = "cd" ]
 	path=`goto show "$2"`
 	cd "$path"
 else
```

Comment:
Please consider reactor to use early return when handling special cases, and then put general case at the end after all special cases have been handled. Refactor example --> https://gist.github.com/Arxcis/7780b55ea94b6f6ca113da0b9db78a90